opencti-platform/opencti-graphql/src/config/conf.js

```diff
@@ -405,22 +405,23 @@ export const getPlatformHttpProxyAgent = (uri) => {
   return undefined;
 };
 
+// General
+export const ENABLED_API = booleanConf('app:enabled', true);
+export const ENABLED_UI = booleanConf('app:enabled_ui', true);
+
 // Playground
 export const ENABLED_DEMO_MODE = booleanConf('demo_mode', false);
-export const PLAYGROUND_INTROSPECTION_DISABLED = DEV_MODE ? false : booleanConf('app:graphql:playground:force_disabled_introspection', true);
-export const PLAYGROUND_ENABLED = booleanConf('app:graphql:playground:enabled', true);
+export const PLAYGROUND_INTROSPECTION_DISABLED = DEV_MODE ? false : (!ENABLED_UI || booleanConf('app:graphql:playground:force_disabled_introspection', true));
+export const PLAYGROUND_ENABLED = ENABLED_UI && booleanConf('app:graphql:playground:enabled', true);
 export const GRAPHQL_ARMOR_ENABLED = booleanConf('app:graphql:armor_enabled', true);
 
 // Default activated managers
-export const ENABLED_API = booleanConf('app:enabled', true);
 export const ENABLED_TRACING = booleanConf('app:telemetry:tracing:enabled', false);
 export const ENABLED_METRICS = booleanConf('app:telemetry:metrics:enabled', false);
-export const ENABLED_TELEMETRY = booleanConf('app:telemetry:filigran:enabled', false);
```

Comment:
These variables were unused ? (`ENABLED_TELEMETRY` and `ENABLED_TELEMETRY_MANAGER`)

Reply:
Looks like. As telemetry cannot be disable, should have been forgotten